ami.tf

```
@@ -7,9 +7,15 @@ locals {
     "AL2_ARM_64" : "-arm64",
     "BOTTLEROCKET_x86_64" : "x86_64",
     "BOTTLEROCKET_ARM_64" : "aarch64"
+    "BOTTLEROCKET_ARM_64_NVIDIA" : "-gpu"
```

do you mind converting this to a normal hcl map?

```suggestion
    BOTTLEROCKET_ARM_64_NVIDIA = "-gpu"
```

---

I updated the lot